### ISSUE TITLE
Training page statistics

### ISSUE THREAD

**Issue by nomadscientist** [galaxyproject/training-material#5014]
Next step for this: https://github.com/galaxyproject/training-material/pull/4984
1) Would be good to get a better description of 'Goal conversions' as idk what that means and I might use it more if I did. Like I think that's the 'I completed the tutorial' button probably? And then maybe that could be broken down across tutorials, and/or compared with "People accessing page vs Completing tutorial"

2) Total tutorials over time (i.e. pull data from 'Date created' and then that can show a nifty graph)

3) Contributors over time for the page (same concept?)

4) So once Workflow testing is there, would also be good to have a 'Last updated' listing for each tutorial, and more specifically it can be organised 

5) We can't sort the listings, i.e. "Top pages" is awesome but I'd like to know the tutorials nobody is accessing (I.e. worst pages), as it might be we need to remove them as they are now antiquated.

6) Incidentally, I can't actually download the data from the various lists as a table. Can a button be added to allow this?

**Comment by nomadscientist** [comment]
7. Oh mate, the 'Event tracking' is absolutely brilliant. It would be SO NICE to know what's happening with this so we can bring trainers together better. Is there an FAQ or something on what we need to do to log an event so that it shows up there?

**Comment by hexylena** [comment]
> Total tutorials over time (i.e. pull data from 'Date created' and then that can show a nifty graph)

so just a subset of the global stats page? https://training.galaxyproject.org/training-material/stats/ yeah doable

most of these are very doable

**Comment by hexylena** [comment]
> Is there an FAQ or something on what we need to do to log an event so that it shows up there?

if an event is registered and it uses one of the single cell training materials it will show up. (so we will not see external events)

**Comment by nomadscientist** [comment]
> > Total tutorials over time (i.e. pull data from 'Date created' and then that can show a nifty graph)
> 
> so just a subset of the global stats page? https://training.galaxyproject.org/training-material/stats/ yeah doable
> 
> most of these are very doable

Yes! The 'Growing community' graphs look amazing

**Comment by nomadscientist** [comment]
What does the 'Comments' column mean in the Workflows?

**Comment by nomadscientist** [comment]
Why is this saying that everything got updated today? That's weird (https://training.galaxyproject.org/training-material/topics/single-cell/maintainer.html)

**Comment by nomadscientist** [comment]
Right, so having reflected, I think that basically this maintainer page could have 2 sections:
1) Impact / metrics (Tools, tutorials, events/news over time cumulatively; updating; a lot of the excellent graphs you've made but now available for everyone writing grants etc.) In an ideal world, contributors over time (but I think that was unfeasible)
and
 - References (and perhaps a section of 'anything that cites those main references?'
2) Maintainer info (what's working on which servers, when it was last updated, is it using the latest tools in the toolshed)

**Comment by nomadscientist** [comment]
Also, there's a bit that says:
Events using materials from this Topic
TODO once this is merged: https://github.com/galaxyproject/training-material/pull/4963
^^but that has been merged now :)

**Comment by nomadscientist** [comment]
Perhaps, in fact, an 'Impact' and a 'Maintainer' page as separate entities?

**Comment by nomadscientist** [comment]
Connecting this here too: https://github.com/galaxyproject/training-material/issues/5486

**Comment by nomadscientist** [comment]
I will also add something that seems related - it would be lovely to have a 'Community' (i.e. the SPOC Community) page that has the following metrics:
- number of members, events, publications, tools, training, news posts, per year

Obviously this means we need to tag stuff effectively, but would be lovely to be able to see that kind of a page too. Probably more of a Hub idea than a GTN idea, so will post it there.
https://github.com/galaxyproject/galaxy-hub/issues/2864

**Comment by nomadscientist** [comment]
Mate, this is fecking brilliant.

But unfortunately for the next person building training infrastructure, this image:
<img width="666" alt="Screenshot 2024-12-19 at 15 32 20" src="https://github.com/user-attachments/assets/0a5495ea-de35-4e87-afb3-d84335ba3262" />

really clarified for me that a 3rd page "Usage page" of updated page/topic metrics as discussed in this and the other linked issue and with the existing page metrics, would be ideal. For the future!
